src/main/java/com/redhat/devtools/intellij/commonuitest/utils/runner/IntelliJ.java

```diff
@@ -0,0 +1,54 @@
+/*******************************************************************************
+ * Copyright (c) 2022 Red Hat, Inc.
+ * Distributed under license by Red Hat, Inc. All rights reserved.
+ * This program is made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution,
+ * and is available at https://www.eclipse.org/legal/epl-v20.html
+ *
+ * Contributors:
+ * Red Hat, Inc. - initial API and implementation
+ ******************************************************************************/
+package com.redhat.devtools.intellij.commonuitest.utils.runner;
+
+/**
+ * Supported versions of the IntelliJ Idea enumeration
+ *
+ * @author zcervink@redhat.com
+ */
+public class IntelliJ {
```

Comment:
You do not need to wrap the enum into java class. you can create enum file with enum directly.

Reply:
Extracted.